pom.xml

```diff
@@ -54,6 +54,7 @@
         <junit-jupiter.version>5.6.0</junit-jupiter.version>
         <junit.version>4.12</junit.version>
         <junit-platform.version>1.0.1</junit-platform.version>
+        <log4j.version>2.17.1</log4j.version>
```

Comment:
This doesn't have any effect if not referenced. We don't need to add this.

Could you instead set that version on this row:
https://github.com/rovio/rovio-ingest/blob/28f56c2e6299ada66e955cdebbdb81968fa3aa3c/pom.xml#L71